systemd/docker-lvm-plugin.service

```diff
@@ -5,8 +5,8 @@ Before=docker.service
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/docker-lvm-volumegroup
-ExecStart=/usr/bin/docker-lvm-plugin
+EnvironmentFile=-/etc/sysconfig/docker-lvm-plugin
```

Comment:
I didn't modify it, I just fixed the suffix (`s/volumegroup/docker-lvm-plugin/`)